frontend/src/components/CurrentBooking.test.tsx

```diff
@@ -1,7 +1,62 @@
+// @ts-nocheck
 import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
 import CurrentBooking from './CurrentBooking';
+import * as bookingService from '../services/bookingService';
+import { getBookings } from '../services/bookingService';
+import userEvent from '@testing-library/user-event';
+import ReactDOM, { unmountComponentAtNode } from 'react-dom';
 
-test('renders CurrentBooking.tsx', () => {
-    expect(true).toBe(true);
+const fakeBooking = [
+    {
+        id: '123',
+        startTime: '2021-10-21T17:32:28Z',
+        endTime: '2021-10-21T19:32:28Z',
+        room: {
+            id: 'c_188fib500s84uis7kcpb6dfm93v25@resource.calendar.google.com',
+            name: 'Amor',
+            capacity: 4,
+            features: ['Jabra', 'TV', 'Webcam'],
+            nextCalendarEvent: '2021-10-21T17:32:28Z'
+        }
+    }
+];
+
+let container = null;
+beforeEach(() => {
+    // setup a DOM element as a render target
+    container = document.createElement('div');
+    document.body.appendChild(container);
+});
+
+afterEach(() => {
+    // cleanup on exiting
+    unmountComponentAtNode(container);
+    container.remove();
+    container = null;
+});
+
+it('renders booking data and checks for booked room name', async () => {
```

Comment:
Yleensä nimettäessä testejä kerrotaan mitä testattavan komponentin pitäisi tehdä vs. mitä testi tekee. Esim:

`it('renders booking data with correct name',...)`
`it('shows capacity after expanding',...)`

Ehkä tääki vähä makuasia.

Reply:
Vaihdettu